docs/howto/operator.rst

```diff
@@ -921,3 +921,92 @@ See `Google Cloud Sql Proxy documentation
 <https://cloud.google.com/sql/docs/postgres/sql-proxy>`_
 for details about Cloud Sql Proxy.
 
+Google Cloud Storage Operators
+------------------------------
+
+GoogleCloudStorageBucketCreateAclEntryOperator
```

Comment:
To me `GoogleCloudStorageCreateBucketAclEntryOperator` sounds bit more descriptive. What do you think?